src/v2/Apps/ViewingRoom/__tests__/ViewingRoomApp.jest.tsx

```diff
@@ -247,8 +318,13 @@ describe("ViewingRoomApp", () => {
     }
     it("shows sign up modal", async () => {
       const wrapper = await getWrapper()
-      const html = wrapper.html()
-      console.log(html)
+      expect(wrapper.find("ViewingRoomMeta").length).toBe(1)
+      expect(wrapper.find("AppContainer").length).toBe(1)
+      expect(wrapper.find("ViewingRoomHeader").length).toBe(1)
+      // expect(wrapper.find("ViewingRoomTabBar").length).toBe(0)
```

Comment:
@ansor4 - that is what I was mentioning I do not understand. From reading code I assumed that that is how page should be laid out if the user is not logged in. But those commented lines fail locally for me. Leaving it commented but removing console.log though.

Reply:
This is definitely weird, so it's rendering tab bar and the content not accessible component? Happy to pair on this, but maybe let's not sink too much time into this.